src/ngraph/op/util/broadcasting.cpp

```diff
@@ -104,6 +104,19 @@ static std::pair<ngraph::Shape, std::vector<ngraph::Shape>>
     return get_numpy_broadcast_shapes(input_shapes);
 }
 
+static std::pair<ngraph::Shape, std::vector<ngraph::Shape>>
```

Comment:
The duplication in this file is a bit of a drag, but I don't have any bright ideas. (Nor even any dim ones.)

Reply:
The vector ones aren't as friendly.

Reply:
FWIW, if `OutputVector` were its own class (is it not already?), we _could_ have an `as_single_output_nodes` akin to what we have for `Output`.

Reply:
It's not.

Reply:
I think this idea is probably a good one, but maybe in a later PR.